### PR TITLE
Unit test feature

### PR DESCRIPTION
**Pull request by stospina**
code review ... thank you

### CODE REVIEW

**Review by florpadilla — APPROVED** [review]
Looks good!

File: CONVERTER-SERVICE/__test__/imageConverter.model.test.js [hunk 1]
```diff
@@ -24,9 +24,15 @@ describe('Test Image Converter', () => {
       false,
       false
     );
+<<<<<<< HEAD
```
Comment (DilanOjeda) [inline CONVERTER-SERVICE/__test__/imageConverter.model.test.js:27]: Remove this line

File: MACHINE-LEARNING-SERVICE/src/model/models/yolo.model.js [hunk 2]
```diff
@@ -31,36 +31,34 @@ class Yolo extends ObjectDetection {
...
-        throw new MachineLearningException('Error building model YOLO.', 'ML-02');
-      });
-
-      const foundObjectsArray = FilerResults.filterFunction(
-        imagesToPredictArray,
-        this.objectRequired,
-        this.percentage
-      );
-      return foundObjectsArray;
+
```
Comment (DilanOjeda) [inline MACHINE-LEARNING-SERVICE/src/model/models/yolo.model.js]: this line is not necessary

File: MACHINE-LEARNING-SERVICE/src/model/models/mobilenet.model.js [hunk 3]
```diff
@@ -30,29 +30,30 @@ class MobileNet extends ObjectDetection {
...
-        throw new MachineLearningException(
-          'Error building model MOBILENET.',
-          'ML-01'
-        );
-      });
-      const foundObjectsArray = FilterResults.filterFunction(
-        imagesToPredictArray,
-        this.objectRequired,
-        this.percentage
+
```
Comment (DilanOjeda) [inline MACHINE-LEARNING-SERVICE/src/model/models/mobilenet.model.js]: This line is not nessary

File: MACHINE-LEARNING-SERVICE/src/model/models/cocoSsd.model.js [hunk 4]
```diff
@@ -28,28 +28,27 @@ class CocoSsd extends ObjectDetection {
...
-        throw new MachineLearningException('Error building model COCO.', 'ML-01');
-      });
-
-      const foundObjectsArray = FilterResults.filterFunction(
-        imagesToPredictArray,
-        this.objectRequired,
-        this.percentage
-      );
-      return foundObjectsArray;
+
```
Comment (DilanOjeda) [inline MACHINE-LEARNING-SERVICE/src/model/models/cocoSsd.model.js]: This line is not necessary

**Review by DilanOjeda — APPROVED** [review]
Terrific!

File: MACHINE-LEARNING-SERVICE/src/model/models/yolo.model.js (hunk 2 above)
Comment (stospina) [reply MACHINE-LEARNING-SERVICE/src/model/models/yolo.model.js]: Done! thank you

**Review by florpadilla — APPROVED** [review]
good!

**Pull request merged**